hive/tests/test_check.py

```diff
@@ -5,6 +5,25 @@
 import pytest
 
 
-@pytest.mark.usefixtures('dd_environment')
-def test():
-    pass
+@pytest.mark.e2e
+def test(dd_agent_check):
+    instance = {}
+    aggregator = dd_agent_check(instance)
+    metrics = [
+        "hive.server.memory.heap.committed",
+        "hive.server.memory.heap.init",
+        "hive.server.memory.heap.max",
+        "hive.server.memory.heap.used",
+        "hive.server.memory.non_heap.committed",
+        "hive.server.memory.non_heap.init",
+        "hive.server.memory.non_heap.max",
+        "hive.server.memory.non_heap.used",
+        "hive.server.memory.total.committed",
+        "hive.server.memory.total.init",
+        "hive.server.memory.total.max",
+        "hive.server.memory.total.used",
+        "hive.server.session.active",
+        "hive.server.session.open",
+    ]
+    for metric in metrics:
+        aggregator.assert_metric(metric)
```

Comment:
Can we add `aggregator.assert_all_metrics_covered()` if it's possible?

Reply:
Then I need to add al jvm metrics to the list, and I dont think they're relevant

Reply:
I think there is a config option not to emit those. Will check

Reply:
Setting 'collect_default_metrics' to False and I still get:
```
Missing metrics: {'jvm.thread_count', 'jvm.non_heap_memory', 'jvm.gc.cms.count', 'jvm.os.open_file_descriptors', 'jvm.cpu_load.process', 'jvm.heap_memory_committed', 'jvm.gc.eden_size', 'jvm.non_heap_memory_max', 'jvm.loaded_classes', 'jvm.non_heap_memory_committed', 'jvm.gc.parnew.time', 'jvm.cpu_load.system', 'jvm.gc.survivor_size', 'jvm.heap_memory', 'jvm.heap_memory_max', 'jvm.non_heap_memory_init', 'jvm.heap_memory_init', 'jvm.gc.old_gen_size'}
```
So I think its better to leave it this way